baseplate/integration/pyramid.py

```diff
@@ -86,18 +86,38 @@ class ServerSpanInitialized(BaseplateEvent):
     pass
 
 
+class HeaderTrustHandlerInterface(object):
+    """Class used by BaseplateConfigurator to validate headers.
+    """
+
+    def should_trust_trace_headers(self, request):
+        raise NotImplementedError
+
+
+class StaticTrustHandler(HeaderTrustHandlerInterface):
```

Comment:
this should probably have a docstring and both this and the base class should probably be added to the `.rst` documentation so it shows up in the generated docs